### PR TITLE
CTOR-555 : Pack(operatingsystems-aix-ssh-storage) : Add usage-prct thresholds

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Update the documentation and add new macros in

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] 25.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-aix-ssh.md [hunk 1]
```diff
@@ -5,206 +5,523 @@ title: AIX SSH
...
-    --ssh-password='centreon-password' \
-    --ssh-backend=sshcli \
-    --filter-type='SVG' \
-	--critical-status='%{state} =~ /stale/i'\
-    --verbose
+	--plugin=os::aix::local::plugin  \
+	--mode=storage \
+	--hostname='10.0.0.1' \
+	--ssh-backend='sshcli' \
+	--ssh-username='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-aix-ssh.md:332]: username & password

File: pp/integrations/plugin-packs/procedures/operatingsystems-aix-ssh.md [hunk 2]
```diff
@@ -5,207 +5,518 @@ title: AIX SSH
...
 <Tabs groupId="sync">
-<TabItem value="Cmdreturn" label="Cmdreturn">
+<TabItem value="OS-AIX-SSH-custom" label="OS-AIX-SSH-custom">
 
-| Metric name                               | Description                            | Unit  |
-| :---------------------------------------- | :------------------------------------- | :---- |
-| command.exit.code.count                   | Number of exit code return             | count |
+| Service Alias | Service Template          | Service Description                             |
+|:--------------|:--------------------------|:------------------------------------------------|
+| Errpt         | OS-AIX-Errpt-SSH-custom   | Check AIX erreur pront by SSH                   |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/operatingsystems-aix-ssh.md:29]: erreur pront??

File: pp/integrations/plugin-packs/procedures/operatingsystems-aix-ssh.md [hunk 3]
```diff
@@ -5,207 +5,518 @@ title: AIX SSH
...
-    --ssh-backend=sshcli \
-    --filter-type='SVG' \
-	  --critical-status='%{state} =~ /stale/i'\
-    --verbose
+	--plugin=os::aix::local::plugin  \
+	--mode=storage \
+	--hostname='10.0.0.1' \
+	--ssh-backend='sshcli' \
+	--ssh-username='' \
+	--ssh-password='' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/operatingsystems-aix-ssh.md:328]: username and password?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-aix-ssh.md [hunk 4]
```diff
@@ -5,206 +5,523 @@ title: AIX SSH
...
+	--ssh-password='' \
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-aix-ssh.md:333]:
```suggestion
	--ssh-username='XXXX' \
	--ssh-password='XXXX' \
```

File: pp/integrations/plugin-packs/procedures/operatingsystems-aix-ssh.md [hunk 5]
```diff
@@ -5,207 +5,518 @@ title: AIX SSH
...
+	--ssh-port=''  \
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/operatingsystems-aix-ssh.md:329]:
```suggestion
	--ssh-password='XXXX' \
	--ssh-port='XXXX'  \
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-aix-ssh.md [hunk 6]
```diff
@@ -5,206 +5,523 @@ title: AIX SSH
...
+
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="OS-AIX-SSH-custom" label="OS-AIX-SSH-custom">
+
+| Alias   | Modèle de service         | Description                                                                         |
+|:--------|:--------------------------|:------------------------------------------------------------------------------------|
+| Errpt   | OS-AIX-Errpt-SSH-custom   | Contrôle permettant de vérifier les erreurs AIX via SSH                             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-aix-ssh.md:30]:
```suggestion
| Errpt   | OS-AIX-Errpt-SSH-custom   | Contrôle permettant de vérifier les messages retournés par la commande AIX 'errpt' via SSH                             |
```

File: pp/integrations/plugin-packs/procedures/operatingsystems-aix-ssh.md (hunk 2 above)
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/operatingsystems-aix-ssh.md:29]:
```suggestion
| Errpt         | OS-AIX-Errpt-SSH-custom   | Check AIX command 'errpt' output messages using SSH                   |
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**